.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -0,0 +1,48 @@
+name: Bug Report
+description: Create a bug report for the MetaMask SDK
+labels: ["template: bug"]
+body:
+  - type: textarea
+    attributes:
+      label: Provide environment information
+      description: "Give us a general overview of the environment where the issue is happening"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: MetaMask Version
```

Comment:
Is this the MetaMask wallet version? We also need them to provide the SDK version